### ISSUE TITLE
Bulk Changing FOIPPA Section - Doesn't Work and App Crashes

### ISSUE THREAD

**Issue by KyEggleston** [bcgov/foi-flow#4376]
**Describe the bug in current situation**
In the pilot user session, a few users raised that there was an issue with bulk changing redaction codes. After trying to bulk change redactions, the changes didn't apply, and then the app crashed. However, this issue was fixed with a refresh. 

This needs to be investigated further to determine cause. 

**Link bug to the User Story**

**Impact of this bug**
Describe the impact, i.e. what the impact is, and number of users impacted. 

Undetermined

**Chance of Occurring (high/medium/low/very low)**

Undetermined

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**

**Steps to Reproduce**

**Actual/ observed behaviour/ results**

**Expected behaviour**
A clear and concise description of what you expected to happen. Use the gherking language. 

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

**Comment by KyEggleston** [comment]
Test after #4196 back to QA

**Comment by nimya-aot** [comment]
@divyav-aot when we do bulk changing FOIPPA Sections, then changes not happening if there is a redaction box inside the other one on the same page.

**Comment by nimya-aot** [comment]
Created a separate ticket for the above mentioned issue. https://app.zenhub.com/workspaces/foi-flow-607f0ab6ae54b700111dc733/issues/gh/bcgov/foi-flow/4553
Closing this one as working as expected.
